packages/api/lambdas/bootstrap.js

```diff
@@ -269,13 +283,15 @@ function handler(event, context, cb) {
     bootstrapElasticSearch(get(es, 'host')),
     bootstrapUsers(get(users, 'table'), get(users, 'records')),
     bootstrapCmrProvider(get(cmr, 'Password')),
+    bootstrapLaunchpad(get(launchpad, 'Passphrase')),
```

Comment:
This returns false if there is no passphrase? Are we sure this will only get called if we're using launchpad auth? 
Don't want it to break if they're still using earthdata login

Reply:
It will return 'nopassphrase' if Passphrase is not set in .env.